snowshu/core/graph.py

```diff
@@ -282,7 +283,8 @@ def _process_downstream_relation_set(
                 f'as they are usually unintended side effects of lenient regex.'
             )
         # check to make sure found upstream relation is not a view
-        view_relations = [r.quoted_dot_notation for r in upstream_relations if r.is_view]
+        adapter = SnowflakeAdapter()
+        view_relations = [adapter.quoted_dot_notation(r) for r in upstream_relations if r.is_view]
```

Comment:
I just noticed this change while looking at the `preserve_case` PR. The core snowshu code shouldn't need to have any idea what the adapter's implementation is.

I don't know why we are looking for the quoted notation here as we are only concerned with the existence of views, not the exact names of them. Probably nice print output.

We can eliminate this use of `quoted_dot_notation` here and the references to the snowflake adapter.